src/mcp/tools/auth/disable_auth_user.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { disableUser } from "../../../gcp/auth.js";
+
+export const disable_auth_user = tool(
+  {
+    name: "disable_auth_user",
+    description: "Disables or enables a user based on a UID.",
+    inputSchema: z.object({
+      uid: z.string().describe("The localId or UID of the user to disable or enable"),
+      disabled: z.boolean().describe("true disables the user, false enables the user"),
+    }),
+    annotations: {
+      title: "Disable or enable a particular user",
+      readOnlyHint: true,
```

Comment:
This isn't readonly - it should probably be marked as dangerous and idempotent. See https://modelcontextprotocol.io/docs/concepts/tools#tool-definition-structure

Reply:
Whoops, good catch